.gitignore

```diff
@@ -21,3 +21,13 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
+
+# OS files
+.DS_Store
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
```

Comment:
Should we use our [maven .gitignore template](https://github.com/companieshouse/styleguides/blob/master/gitignore/maven)?

Reply:
Check the latest changes please

Reply:
I'm not sure whether we need to keep the entries currently in master. Could we just override it with the template?

Reply:
done!